### PR TITLE
add audit logs

### PR DESCRIPTION
**Pull request by LeoCavaille**

### CODE REVIEW

File: assets/audit-logs/policy.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# extended audit of auth delegation
+- level: RequestResponse
+  omitStages:
+  - RequestReceived
+  resources:
+  - group: authorization.k8s.io
+    resources:
+    - subjectaccessreviews
+
+# NP - NG
```
Comment (roboll) [inline assets/audit-logs/policy.yaml]: Should we drop this part? Not relevant to the workshop.

File: katacodacomleocdd-k8s-monitoring/background.sh [hunk 2]
```diff
@@ -1,3 +1,14 @@
 git clone https://github.com/LeoCavaille/k8s-workshop
 mv k8s-workshop/assets ./assets
 rm -rf k8s-workshop
+
+# add audit logs to the apiserver
+mkdir -p /etc/kubernetes/audit-policies
+cp assets/audit-logs/policy.yaml /etc/kubernetes/audit-policies/policy.yaml
+
+sed -i '/tls-private-key-file/a \ \ \ \ - --audit-policy-file=/etc/kubernetes/audit-policies/policy.yaml' /etc/kubernetes/manifests/kube-apiserver.yaml
```
Comment (roboll) [inline katacodacomleocdd-k8s-monitoring/background.sh:9]: 🙈

Comment (LeoCavaille) [reply katacodacomleocdd-k8s-monitoring/background.sh:9]: 🥇

File: assets/audit-logs/policy.yaml (hunk 1 above)
Comment (LeoCavaille) [reply assets/audit-logs/policy.yaml]: 👍 just copy/pasted ours will update

**Review by roboll — APPROVED** [review]

**Pull request merged**